File: Java/UberX.java
package Java;

class UberX extends Car {
    String brand;
    String model;

    public UberX(String license, Account driver, String brand, String model) {
        //? "super" makes reference to atributes and methods of the super class or father class
        super(license, driver);
        //? While "this" makes reference to the class you are right know or the child class
        this.brand = brand;
        this.model = model;
    }
}

File: PHP/uberX.php
<?php
require_once('./car.php');
class UberX extends Car {
    public $brand;
    public $model;

    public function __construct($license, $driver, $brand, $model){
        parent::__construct($license,$driver);
        $this->brand = $brand;
        $this->model = $model;
    }
    //? frunction inherited from printDataCar in Car.php
    public function printDataCar() {
        parent::printDataCar(); //  This shows the model and the brand of an UberX
        echo"<br>
        Modelo: $this->model
        Marca: $this->brand
        ";
    }
}
?>